dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.telemetry-1.1.ee-7.0.feature

```diff
@@ -0,0 +1,20 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpTelemetry-1.1.ee-7.0
+singleton=true
+-features=\
+  com.ibm.websphere.appserver.javax.annotation-1.3, \
```

Comment:
why specify the annotation version here? wouldn't it be better to just use the version that CDI pulls in?